Content.Shared/Anomaly/Effects/Components/ElectricityAnomalyComponent.cs

```diff
@@ -1,53 +1,59 @@
-﻿using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
 
 namespace Content.Shared.Anomaly.Effects.Components;
 
 [RegisterComponent]
 public sealed partial class ElectricityAnomalyComponent : Component
 {
+    /// <summary>
+    /// the number of lightning strikes, at the maximum severity of the anomaly
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public int MaxBoltCount = 3;
```

Comment:
just from the vid you posted, this feels kinda weak. perhaps bump it up to like 5 max so containment isn't as trivial as throwing down a single grounding coil or whatever.

Reply:
Well, now the electrical anomaly when appearing in a random place of the station will strike lightning in the power devices, and can even explosions to arrange depressurization. But it will be from the sound of explosions much easier to find. To secure the content of the anomaly must buy in the cargo 3 tesla coils or grounding rods, and bring to the anomaly.

Reply:
increasing the maximum number of lightning bolts to 5, will require to bring respectively 5 tesla coils or grounding rods

Reply:
though, the amount of lightning is multiplied by the severity of the anomaly, which means only 1-2 shots at the very beginning of growth

Reply:
i think that's fine. the lightning is a pretty major part of its danger so needing more than a few coils at low severity is fine. It might even make sense to have a min amount of lightning and lerp the value so that there's never less than 2 or something.

Reply:
Done